rai/node/node.cpp

```diff
@@ -863,7 +863,30 @@ void rai::vote_processor::vote (std::shared_ptr<rai::vote> vote_a, rai::endpoint
 	std::lock_guard<std::mutex> lock (mutex);
 	if (!stopped)
 	{
-		if (votes.size () < 256 * 1024)
+		bool process (rai::rai_network == rai::rai_networks::rai_test_network);
```

Comment:
This initial value doesn't do anything as it'll be overridden later. I'd put this condition as the first if statement, and keep the rest as "else if"s.

Reply:
I guess it does help if the rep is >= 5%. Was that intended?

Reply:
Test network value required only for votes processor profiling, good catch